### ISSUE TITLE
Do not mask the menu icons. 

### ISSUE THREAD

**Issue by MarcBrillault** [MarcBrillault/twitter-anonymizer#10]
v1.0.1 introduced a very broad selector of links which masks all the menu items.

We still need to hide the identity below the menu.

A `nav [aria-label="Principal"]` to make everything visible should be enough.
